src/signalcorr.jl

```diff
@@ -89,10 +103,17 @@ autocov{T<:Real}(x::VecOrMat{T}; demean::Bool=true) = autocov(x, default_autolag
 
 ## autocor
 
+"""
+    autocor!(r, x, lags; demean=true)
+
+Compute the autocorrelation function (ACF) of a vector or matrix `x` at `lags`
```

Comment:
As far as I know, yes. I took a course on time series in grad school and we always referred to the autocorrelation in this way as the ACF (likewise for PACF for the partial autocorrelation). Incidentally, R's equivalent function to `autocov` is `acf`; I was actually a little surprised that Julia didn't follow this convention, particularly since `pacf` matches the corresponding function in R.